proposal.md

```diff
@@ -44,6 +58,27 @@ the project requires a contributor license agreement or other
 procedural steps, please explain here.  "None at this time" is
 acceptable for this question.
 
+* The Clawpack community was most active approximately 5-10 years ago and is less active these days. It may not be the best choice going forward for a project.
+
+    * PyClaw has the most contributors and lots of documentaiton (would be the best solver of the four to choose for a project).
+    * GeoClaw had the most changes in the past year, but only one contributor.
+    * Submitted request to join Clawpack Slack channel but have yet to hear back from them.
+    
+* Travis continuos integration is used (service used to build and test software projects hosted on GitHub and Bitbucket).
+
+* If using either the Fortran or Python installations there is an automated test suite that runs several tests and compares the solution (parts of the soltution) to archived resutls.
+
+    * For PyClaw, 46 of 48 tests in the test suite pass and two tests fail. The two tests failed with the error "no such file or directory" for the verification file (the file name for the verificaiton file might be mislabeld for those tests).
+    * For the calssic solver, all three tests in the test suite failed.
+
+* Permissive license: Berkeley Software Distribution (BSD) license (BSD 3-Clause)
+    * Concern about license: On the Clawpack website the copyright is from 1994-2020, does that mean it is no longer in effect? 
+        * "Copyright (c) 1994–2020, The Clawpack Development Team. All rights reserved."
```

Comment:
Copyright exists whether you write that line or not, for every year you develop it, also whether or not it's written. I personally dislike including the year because it leads to this sort of confusion. Some projects use automation to update the copyright notice on every file each year (but that's technically incorrect if the file hasn't been modified). The fact that copyright nominally has a finite duration seems moot when Congress keeps extending the expiration to appease Disney, which lobbies to protect its characters from entering the public domain.